serverless/main.mk

```diff
@@ -1,26 +1,39 @@
 # Macroses
 ########################################################################################################################
+PROJ_PATH ?= $(PROJ_DIR)/$(SLS_SERVICE)-$(SLS_FUNCTION)
```

Comment:
This is somewhat shadowing the `PROJECT_PATH` variable. Becomes confusing. Maybe if we use `serverless invoke`, we can switch back to `PROJECT_PATH`?

Reply:
fixed